caddy.go

```diff
@@ -74,6 +74,9 @@ var (
 	// false after that.
 	started bool
 
+	// record where the process log is being sent
+	LogDestination string
```

Comment:
Is this supposed to be a filename?

Reply:
Also, does the `caddy` package ever actually honor this variable? It seems arbitrary to store it in this package when `package main` is the only one to set it.

Reply:
LogDestination is what is passed in by the `-log` parameter.  Can be a filename or `stdout` etc

I'm not sure what you mean by honor the variable?  It is set as a flag and read in the main function but needed in the function in server.go to indicate if `-log` is being written to `stdout` or not.  Perhaps you could suggest a better place for it.

Reply:
Sorry, by "honor the variable" I basically meant "use it".

Let's start with this instead: is eliminating the duplication is really that important? This smells like unneeded complexity, for virtually no functional gain.

Reply:
The only gain that warrants this change is that we can check it from within `server.go`. I personally thought it'd be fine but did tag you in on it on the conversation discussing it (but we got a bit hasty and merged before you had opportunity to address it, naturally). I couldn't think of an alternative to be able to determine if the log output was `stdout` for the purpose of cancelling out double-up output.